### PR TITLE
Fix errors in CHANGELOGS for slack and amazon

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
cc: @Taragolis -> FYI I thought I have protection against it (running twine checks) but apparently It did not work. Some of the Sphinx directtives cannot be used in CHANGELOG as it is part of the PyPI package.

**Comment by Taragolis** [comment]
> cc: @Taragolis -> FYI I thought I have protection against it (running twine checks) but apparently It did not work. Some of the Sphinx directtives cannot be used in CHANGELOG as it is part of the PyPI package.

That is a bit strange because I could build it locally without any error if run
`breeze build-docs --package-filter apache-airflow-providers-amazon` and result rendered correctly

Result from my last local build
image

**Comment by Taragolis** [comment]
Sorry got it... miss **as part of PyPI package**

**Comment by potiuk** [comment]
Yeah. Seems we lost the twine check during provider package testing in CI (it was there) while converting bash to Python.

Adding it back in https://github.com/apache/airflow/pull/26748

File: airflow/providers/slack/CHANGELOG.rst [hunk 1]
```diff
@@ -30,11 +30,10 @@ Changelog
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
-* The hook class  :class:`airflow.providers.slack.hooks.slack_webhook.SlackWebhookHook` not inherit from
-  :class:`airflow.providers.http.hooks.http.HttpHook` anymore. In practice the only impact on
-  user-defined classes based on **SlackWebhookHook** and you use attributes from **HttpHook**.
-* Drop support deprecated ``webhook_token`` parameter in
-  :ref:`Slack Incoming Webhook Connection <howto/connection:slack-incoming-webhook>` Extra.
+* The hook class  ``SlackWebhookHook`` dies not inherit from ``HttpHook`` anymore. In practice the
```
Comment (eladkal) [inline airflow/providers/slack/CHANGELOG.rst:33]:
```suggestion
* The hook class  ``SlackWebhookHook`` does not inherit from ``HttpHook`` anymore. In practice the
```

**Review by eladkal — APPROVED** [review]

**Pull request merged**